packages/block-editor/src/components/block-navigation/style.scss

```diff
@@ -16,24 +16,6 @@ $tree-item-height: 36px;
 	margin: 0;
 }
 
-.block-editor-block-navigation__list .block-editor-button-block-appender {
```

Comment:
Is this not needed at all?

Reply:
Some of it was, yes, good catch. Most of it wasn't, though.

<img width="613" alt="Screenshot 2020-03-04 at 14 26 16" src="https://user-images.githubusercontent.com/1204802/75883933-23c26f00-5e24-11ea-8063-b192d0d3e64e.png">